app/models/govspeak_decorator.rb

```diff
@@ -0,0 +1,28 @@
+class GovspeakDecorator < DelegateClass(Govspeak::Document)
```

Comment:
Can I suggest we introduce a `decorator` dir rather than mix with the `AR` models.

Reply:
I've added this to decorator folder